infrastructure.tf

```diff
@@ -25,22 +25,22 @@ variable master_memory {
     description = "The amount of memory available for the shape of each master node, in gigabytes. The default value is 16. "
 }
 variable master_boot_size {
-    default     = 500
+    default     = 1024
```

Comment:
Do you have the difference of results (opct counters) and costs from previously 512G 60VPU and this new default?

Reply:
Yes, we do have results from 512G 60VPU. We were seeing the intermittent etcd latency spiking to > 500 ms and by increasing the boot volume, it has stabilized.

Reply:
The cost difference from 500Gb/60VPUs -> 1024GB/90VPUs is $120/month * 3 master nodes = $360/month increase. The size for OCI nodes would need to be higher than other CSPs to achieve similar disk performance because the disk performance scales with disk size in OCI.